index.html

```diff
@@ -5124,25 +5125,28 @@ <h2 id="AudioWorkletNodeOptions">
                 unsigned long numberOfInputs
               </dt>
               <dd>
-                Same to <a>numberOfInputs</a> in <a>AudioNode</a>.
+                Same to <a>numberOfInputs</a> in <a><code>AudioNode</code></a>.
```

Comment:
"Same to" -> "Same as".  But it's not really the same.  It's the value to be assigned to.  Not sure on the best way to phrase that.